website/content/en/docs/kubebuilder/references/client.md

```diff
@@ -0,0 +1,574 @@
+...
+```
+
+**Note**: defaults are set by `client.New` when Options are empty. The default [scheme][code-scheme-default] will have the [core][doc-k8s-core] Kubernetes resource types registered. The caller *must* set a scheme that has custom operator types registered for the new Client to recognize these types.
+
+Creating a new Client is not usually necessary nor advised, as the default Client is sufficient for most use cases.
+
+### Reconcile and the Client API
+
+A Reconciler implements the [`reconcile.Reconciler`][doc-reconcile-reconciler] interface, which exposes the Reconcile method. Reconcilers are added to a corresponding Controller for a Kind; Reconcile is called in response to cluster or external Events, with a `reconcile.Request` object argument, to read and write cluster state by the Controller, and returns a `reconcile.Result`. SDK Reconcilers have access to a Client in order to make Kubernetes API calls.
+
+**Note**: For those familiar with the SDK's old project semantics, [Handle][doc-osdk-handle] received resource events and reconciled state for multiple resource types, whereas Reconcile receives resource events and reconciles state for a single resource type.
+
+```Go
+// ReconcileKind reconciles a Kind object
+type ReconcileKind struct {
+	// client, initialized using mgr.Client() above, is a split client
+	// that reads objects from the cache and writes to the apiserver
+	client.Client
+
+    Log    logr.Logger
+
+	// scheme defines methods for serializing and deserializing API objects,
+	// a type registry for converting group, version, and kind information
+	// to and from Go schemas, and mappings between Go schemas of different
+	// versions. A scheme is the foundation for a versioned API and versioned
+	// configuration over time.
+    Scheme *runtime.Scheme
+}
+
+// Reconcile watches for Events and reconciles cluster state with desired
+// state defined in the method body.
+// The Controller will requeue the Request to be processed again if an error
+// is non-nil or Result.Requeue is true, otherwise upon completion it will
+// remove the work from the queue.
+func (r *ReconcileKind) Reconcile(request reconcile.Request) (reconcile.Result, error)
+```
+
+Reconcile is where Controller business logic lives, i.e. where Client API calls are made via `ReconcileKind.client`. A `client.Client` implementer performs the following operations:
+
+#### Get
+
+```Go
+// Get retrieves an API object for a given object key from the Kubernetes cluster
+// and stores it in obj.
+func (c Client) Get(ctx context.Context, key ObjectKey, obj runtime.Object) error
+```
+**Note**: An `ObjectKey` is simply a `client` package alias for [`types.NamespacedName`][doc-types-nsname].
+
+Example:
+```Go
+import (
+	"context"
+	"github.com/example-org/app-operator/pkg/apis/cache/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func (r *ReconcileApp) Reconcile(request reconcile.Request) (reconcile.Result, error) {
```

Comment:
Update for new scaffold. `request reconcile.Request`==>`req ctrl.Request`
Update the imports as well `"sigs.k8s.io/controller-runtime/pkg/reconcile"`==>`ctrl "sigs.k8s.io/controller-runtime"`